aws/src/main/java/org/apache/iceberg/aws/glue/GlueTableOperations.java

```diff
@@ -322,6 +322,7 @@ void persistGlueTable(
                       .name(tableName)
                       .tableType(GLUE_EXTERNAL_TABLE_TYPE)
                       .parameters(parameters)
+                      .description(glueTable.description())
```

Comment:
Looks good @aajisaka this is great!

Recently we added support for setting the glue description via the `comment` table property in #9530. What are your thoughts on leveraging this table property to set the Glue description. This would allow a user to perform an ALTER statement to update the glue description as well. 

```
ALTER TABLE prod.db.sample SET TBLPROPERTIES (
    'comment' = 'A table description.'
)
```

cc: @amogh-jahagirdar

Reply:
I think the `comment` should be honored. I'll update the patch to

1. If `comment` exists, `comment` is applied.
1. If `comment` doesn't exist, the existing description is retained.